templates/master/00-master/_base/files/etc-kubernetes-manifests-etcd-member.yaml

```diff
@@ -20,7 +20,7 @@ contents:
         - "--output-file=/run/etcd/environment"
         - "--v=4"
         securityContext:
-          priviledged: true
+          privileged: true
```

Comment:
Since we were working fine before this change, and so we didn't need security, I need to know why we don't just remove this section since we don't need privileged to do actions in this directory (or, if we do, how we were able to work).

Reply:
Isn't this because etcd is running as a static pod, it has the privileges to use `hostPath`?